packages/pipeline-editor/src/CustomFormControls/NumberControl.tsx

```diff
@@ -95,11 +116,9 @@ function NumberControl({
           // Don't use type="number" see: https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/
           type="text"
           inputMode="numeric"
-          value={value ?? ""}
+          value={renderedValue}
           onChange={handleChange}
-          onWheel={(e) => {
-            (e.target as any).blur();
-          }}
+          onBlur={handleBlur}
```

Comment:
Why is `onWheel` removed / `onBlur` added? And why is it setting `localValue` to undefined on blur?

Reply:
Nevermind saw the explanation in the issue!